src/test/java/com/commercetools/project/sync/util/TestUtils.java

```diff
@@ -46,9 +46,12 @@ public static void assertAllSyncersLoggingEvents(
     assertProductSyncerLoggingEvents(syncerTestLogger, numberOfResources);
     assertInventoryEntrySyncerLoggingEvents(syncerTestLogger, numberOfResources);
     assertCartDiscountSyncerLoggingEvents(syncerTestLogger, numberOfResources);
+    assertStateSyncerLoggingEvents(
+        syncerTestLogger,
+        numberOfResources + 1); // 2 states as 1 state is built-in and it cant be deleted
```

Comment:
I don't understand this part of this comment. `2 states` Can you re-write it in a different way?

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/133/commits/4e8121ca7625da8c3c6c9ea64d9965bce471cbec